pydy/tests/test_utils.py

```diff
@@ -18,3 +22,61 @@ def test_sympy_equal_to_or_newer_than():
     if parse_version(SETUPTOOLS_VERSION) >= parse_version('8.0'):
         with assert_raises(ValueError):
             sympy_equal_to_or_newer_than('0.7.7', '0.7.6-git')
+
+def test_mass():
+
+    mass = Mass()
+    assert mass.symbol == Symbol('m')
+    assert mass.value == 1
```

Comment:
How is this attribute used? Everything you are designing should be able to output fully symbolic eoms. Our current method is to pass in numerical values to system as a `sys.constants = {}`.